src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/LocalDebuggerCommandArgumentsValueProvider.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.ProjectSystem.Build;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using System;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties
+{
+    [ExportInterceptingPropertyValueProvider("LocalDebuggerCommandArguments", ExportInterceptingPropertyValueProviderFile.UserFileWithXamlDefaults)]
+    internal sealed class LocalDebuggerCommandArgumentsValueProvider : InterceptingPropertyValueProviderBase
+    {
+        private readonly Lazy<ProjectProperties> _projectProperties;
+        private readonly Lazy<IOutputGroupsService, IAppliesToMetadataView> _outputGroups;
+        private readonly ConfiguredProject _configuredProject;
+
+        [ImportingConstructor]
+        public LocalDebuggerCommandArgumentsValueProvider(Lazy<ProjectProperties> projectProperties, 
+                                                          [Import(AllowDefault = true)] Lazy<IOutputGroupsService, IAppliesToMetadataView> outputGroups,
```

Comment:
I'm not sure, but for the OutputKeys stuff I copied directly from the CPS LocalWindowsDebuggerService implementation, and they have it marked as optional there as well.